docs/_documentations/remote-deploying-codewind.md

```diff
@@ -39,11 +39,12 @@ For more information, select from the following IDEs:
   - OpenShift 3.11
   - OpenShift 4.3
   - OpenShift in IBM Public Cloud
-  - Standalone Kubernetes
+  - Stand-alone Kubernetes
   - Kubernetes in Docker
-- However, Codewind runs on any Kubernetes distribution that meets the following requirements:
-  - Includes Kube 1.11 or later
-  - Supports RWX volumes
+- However, Codewind can run on any Kubernetes distribution if the following requirements are met:
```

Comment:
I think the previous section was a little more clear, we just needed to add `Nginx ingress controller installed` to it.